Section_II/competition_and_trophies.tex

```diff
@@ -274,53 +274,107 @@
+      \added{Any player of} the team taking the
+      \added{game interruption}\removed{free kick} may announce that the player
+      is ready to take the \added{game interruption}\removed{free kick} at any
+      point.
       Opponent robots must move to a position at least 0.75 m for KidSize and
-      \removed{TeenSize and} 1.5m for AdultSize away from the ball.
-      They are guaranteed at least 15 seconds to move away from the ball.
-      They may take up to 30 seconds if the team taking the free kick has not
-      announced their robot is ready to take the kick off.
-\item The assistant referee announces that 15 seconds and, if applicable, 30 seconds are over.
-\item The referee may decide to execute the free kick any time between 15 and 30 seconds
-      have passed depending on the call of the team taking the free kick and the position of
-      the opponent robots.
-      The referee may also allow the free kick to be executed before 15 seconds have passed
-      if the team taking the free kick have announced their robot is ready and if no opponent is
-      illegally positioned.
+      1.5m for AdultSize away from the ball.
+      They are guaranteed \removed{at least }15 seconds to move away from the ball.
+      \removed{They may take up to 30 seconds if the team taking the
+      free kick has not
+      announced their robot is ready to take the kick off.}
+\item \removed{The assistant referee announces that 15 seconds and, if applicable, 30 seconds are over.}
+\item The referee \added{announces that the game interruption is prepared after}%
+      \removed{may decide to execute the free kick any time between} 15 \removed{and 30}
+      seconds have passed\removed{ depending on the call of the team taking
+      the free kick and the position of the opponent robots}.
+
+      The referee may also \added{announce that the game interruption is
+        prepared}%
+      \removed{allow the free kick to be executed} before 15 seconds have passed
+      if the team taking the \added{game interruption}\removed{free kick} has
+      announced their robot is ready and if no opponent is illegally positioned.
 \item Any opponent robot still illegally positioned is considered as an
       incapable player and must be removed from the field for 30 seconds
       removal penalty.
-\item When the referee decides to execute the free kick and all opponent robots are
+\item When the referee decides to execute the \added{game interruption}%
+      \removed{free kick} and all opponent robots are
       legally positioned or have been removed from the field,
-      the referee blows the whistle.
-      Then the assistant referee who operates the game controller clicks on
-      "Execute Direct / Indirect Free Kick" Blue/Red.
-      The team that was awarded the free-kick can kick now.
+      \added{the referee announces that the game interruption can be executed to
+        the GameController}.%
```

Comment:
Add: "or the assistant referee operating the game controller. In case of a physical competition, the referee additionally blows the whistle."